docs-ref-conceptual/reference-types-and-status.md

```diff
@@ -57,34 +57,34 @@ az extension list-available --output table
 
 ### Extension
 
-Extensions aren't shipped as part of the CLI but run as CLI commands. Some extensions are a permanent part of the Azure CLI, but often, an extension will give you access to private preview and experimental commands. A single reference group, such as **az iot hub**, can have both core and extension commands.  Here are two examples:
+Extensions aren't shipped as part of the CLI but run as CLI commands. Some extensions are a permanent part of the Azure CLI, but often, an extension gives you access to preview and experimental commands. A single reference group, such as **az iot hub**, can have both core and extension commands.  Here are two examples:
 
 |      Full reference command       | Is Core | Is Extension |
 | --------------------------------- | ------- | ------------ |
 | az iot hub list                   | yes     |              |
 | az iot hub job list               |         | yes          |
 
-You will be prompted to install an extension upon first use.  You can also install an extension by running the [az extension add](/cli/azure/extension#az-extension-add) command.
+You're prompted to install an extension upon first use.  You can also install an extension by running the [az extension add](/cli/azure/extension#az-extension-add) command.
 
 You can learn more about extension references including installation and updating in [Use extensions with the Azure CLI](azure-cli-extensions-overview.md).  See [Available extensions for the Azure CLI](azure-cli-extensions-list.md) for a complete list of extension reference commands.
 
 ## What is reference status?
 
-Regardless of reference type, Azure CLI references fall into three status categories: **GA** (Generally Available), **public preview** or **experimental**. It is the reference command status, not type, that determines stability and support level.
+Regardless of reference type, Azure CLI references fall into three status categories: **GA** (Generally Available), **public preview** or **experimental**. It's the reference command status (not type) that determines stability and support level.
 
 | | GA  | Public preview | Experimental
 |-|-|-|-|
-| **Stability** | Permanent | Can change in response to customer feedback. Is subject to the terms of [Microsoft Azure Previews](https://azure.microsoft.com/support/legal/preview-supplemental-terms/). | Can change in response to customer feedback. Will often migrate to public preview.  Can be removed.
+| **Stability** | Permanent | Can change in response to customer feedback. Is subject to the terms of [Microsoft Azure Previews](https://azure.microsoft.com/support/legal/preview-supplemental-terms/). | Can change in response to customer feedback. Can migrate to public preview.  Can be removed.
```

Comment:
@daphnemamsft , If Acrolinx doesn't like `will often` we need to find a different word that indicates time.  Try `Often migrates`

Reply:
Ok, I have changed the wording